src/build.py

```diff
@@ -112,15 +112,25 @@
 
 # Sync OCaml from a cached tar file because the upstream repository is only
```

Comment:
This comment is now slightly stale. Should probably mention node as a dependency for this url